### PR TITLE
Removed the slugify dependency

### PR DESCRIPTION
**Pull request by moorepants**
According to the upstream requirements.txt this depends on `python-slugify` not `slugify` which are two different packages on PyPi. This causes a `ClobberWarning` when both are installed. See https://github.com/conda-forge/python-slugify-feedstock/issues/33 for more information on the `ClobberWarning`.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by moorepants** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
@conda-forge/sphinx-material please consider merging this: it's likely no version of this theme is presently installable.

**Comment by moorepants** [comment]
Can this be merged? sphinx material can no longer be installed with conda.

**Comment by FaustinCarter** [comment]
@mxr-conda @ndmaxar @oblute @rluria14 Can someone please review this and merge it in? Sphinx-material theme is completely broken right now due to the double dependency.

FWIW, someone from my org would be willing to be added as a feedstock maintainer here since we depend on this pretty heavily internally (if y'all are open to that).

**Comment by bollwyvl** [comment]
We're _well_ over the week threshold of radio silence from @conda-forge/sphinx-material, at this point. 

This means someone who cares about this package can:

- make a [bot command issue](https://github.com/conda-forge/sphinx-material-feedstock/issues/new?assignees=&labels=&template=2-bot-commands.yml) to `@conda-forge-admin, please add user @username` themselves
- ask `@conda-forge/core` to merge it (linking to this PR)
- merge this PR

As I _don't_, particularly, care about this feedstock, I'm unsubscribing: good luck!

**Comment by moorepants** [comment]
@conda-forge/core can you merge this and also add us as maintainers? Thanks.

**Comment by wolfv** [comment]
Happy to merge if you add yourself as maintainer :)

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,3 +47,5 @@ extra:
     - ndmaxar
     - oblute
     - rluria14
+    - moorepants
+
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by moorepants** [comment]
Thanks.

**Pull request merged**